src/actions/CreateWalletActions.tsx

```diff
@@ -136,7 +135,9 @@ export function createAccountTransaction(
           if (error) {
             console.log(error)
             setTimeout(() => {
-              Alert.alert(lstrings.create_wallet_account_error_sending_transaction)
+              Airship.show(bridge => <ButtonsModal bridge={bridge} message={lstrings.create_wallet_account_error_sending_transaction} buttons={{}} />).catch(
```

Comment:
I talked to Madison about these `buttons={}` cases, and she agrees that it looks really weird. We should replace these cases with `buttons={{ ok: { label: lstrings.string_ok_cap } }}` or similar. There is no `onPress` handler needed - the modal just resolves to `'ok' | undefined` instead of `undefined` (not that we care either way).